manage_breast_screening/record_a_mammogram/views.py

```diff
@@ -107,12 +110,35 @@ def get_appointment(self):
 
         return Appointment.objects.get(pk=id)
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
```

Comment:
The views are getting quite messy now. I'd like to do a follow up PR next week that extracts presenter classes or functions.